app/services/stat_retrieval_service.rb

```diff
@@ -21,7 +21,8 @@ def stats_string(user)
 
     def rankings(limit: 1000)
       ranked_users = []
-      User.where('rank != 1500').order('rank desc').limit(limit).each do |u|
+      User.order('rank desc').limit(limit).each do |u|
+        next if u.unranked?
```

Comment:
Doesn't this mean we are not guaranteed to actually get back `limit` results? e.g. if we are displaying the top 15 players and one of them is unranked, then we end up only displaying the top 14